### PR TITLE
Updated 404 page; cleaned up .btn styling

### LINKED ISSUE #70: Update the 404 page
Update the 404 to match the webmaker-desktop design.

CC: @cassiemc @thisandagain

### PR DESCRIPTION
**Pull request by mmmavis**
This fixed #70

### CODE REVIEW

**Comment by mmmavis** [comment]
I added some extra copy.  @alicoding can you also help check to see if I used `react-intl` right?  :smile_cat:

File: src/locales/en-US.yaml [hunk 1]
```diff
@@ -18,6 +18,8 @@ en-US:
   thimble_goggles: Thimble & X-Ray Goggles
   give_us_a_shout: Give us a shout
   give_us_a_shout_sentence: "{GiveUsAShoutLink} if you think there’s something wrong."
+  teach_site_url: teach.mozilla.org
```
Comment (alicoding) [inline src/locales/en-US.yaml]: Make this URL part of your code not in this file.

**Comment by alicoding** [comment]
Just one comment for l10n related stuff

**Comment by mmmavis** [comment]
@alicoding oh yea u r right, since it doesn't need to be translated!  heh I was just blindly following your code example :stuck_out_tongue_closed_eyes:  PR updated again!

**Comment by vazquez** [comment]
I made a PR on your branch @mmmavis 

https://github.com/mmmavis/webmaker-browser/pull/1

R+! :dango:

**Comment by mmmavis** [comment]
YAYYYYY MERGING THIS NOW :dancers: thanks @vazquez and @alicoding

**Pull request merged**